Mergin/projects_manager.py

```diff
@@ -351,7 +371,14 @@ def sync_project(self, project_dir, project_name=None):
             if isinstance(dlg.exception, LoginError):
                 login_error_message(dlg.exception)
             elif isinstance(dlg.exception, ClientError):
-                QMessageBox.critical(None, "Project sync", "Client error: " + str(dlg.exception))
+                if dlg.exception.http_error == 400 and "Another process" in dlg.exception.detail:
+                    # To note we check for a string since error in flask doesn't return server error code
+                    msg = dlg.exception.detail
```

Comment:
What is the message from the server? On the mobile app we say "Somebody else is syncing..". Can we here say something similar like `Somebody else is syncing, please try again later.` please?

Reply:
The message from server is "Another Process is running. Please try later"

Agreed it could be more explicit

Reply:
I change the error message to `Somebody else is syncing, please try again later` to match more closely the mobile app